test/helpers/utils.go

```diff
@@ -699,3 +699,15 @@ func DualStackSupportBeta() bool {
 
 	return GetCurrentIntegration() == "" && supportedVersions(k8sVersion)
 }
+
+// CiliumEndpointSliceFeatureEnabled returns true only if the environment has a kubernetes version
+// equal to 1.21.
+func CiliumEndpointSliceFeatureEnabled() bool {
+	k8sVersionGreaterThan120 := versioncheck.MustCompile(">=1.21.0")
```

Comment:
@tklauser the expression says `>= 1.21` but the name is `greater than 1.20`, ie `>`. I think these are correct, though surprising at first glance since the numbers don't line up.

Reply:
Updated code to reflect the same numbers.

Reply:
@joestringer thanks for pointing that out, I misread the condition.

Maybe we could rename it to something like `k8sVersionGreaterEqual121` to make it match the condition exactly and avoid confusion?